ui/src/app/applications/components/applications-sync-panel/applications-sync-panel.tsx

```diff
@@ -2,12 +2,13 @@ import {ErrorNotification, FormField, NotificationType, SlidingPanel} from 'argo
 import * as React from 'react';
```

Comment:
As discussed, we should look at `application-sync-panel` as well.

Reply:
Will include this `application-sync-panel` in the redesign issue. Thanks for the catch 👍